clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -435,7 +435,9 @@ static KafkaAdminClient createInternal(AdminClientConfig config, TimeoutProcesso
                 .timeWindow(config.getLong(AdminClientConfig.METRICS_SAMPLE_WINDOW_MS_CONFIG), TimeUnit.MILLISECONDS)
                 .recordLevel(Sensor.RecordingLevel.forName(config.getString(AdminClientConfig.METRICS_RECORDING_LEVEL_CONFIG)))
                 .tags(metricTags);
-            reporters.add(new JmxReporter(JMX_PREFIX));
+            JmxReporter jmxReporter = new JmxReporter(JMX_PREFIX);
+            jmxReporter.configure(config.originals());
```

Comment:
How does this work with prefixes?  For example, Kafka Connect passes all config variables with a certain prefix to its admin client.  Does `originals` show those variables as prefixed or not here?

Reply:
my understanding is that originals are the values as originally passed to the admin client. Those should already have prefixes stripped already, since the line above (`config.getString(AdminClientConfig.METRICS_RECORDING_LEVEL_CONFIG)`) does not add any kind of prefixes.